Sources/Utilities/Git.swift

```diff
@@ -75,15 +75,18 @@ public struct GitURL {
 
     public static func from(_ gitURL: String) -> GitURL? {
         do {
-            let regex = try NSRegularExpression(pattern: #"([^/@]+)[:/]([^:/]+)/([^/.]+)(\.git)?$"#, options: .caseInsensitive)
+            let regex = try NSRegularExpression(pattern: #"([^/@]+)[:/]([^:/]+)/([a-z0-9._-]+)$"#, options: .caseInsensitive)
```

Comment:
I am a little worried that this is too restrictive. How about we drop the `.git` suffix first thing then update this regex to match everything after the last `/`?

Reply:
Agreed. Updated with latest push.